common/reviews/api/rush-lib.api.md

```diff
@@ -777,7 +777,7 @@ export class RushConfigurationProject {
     // @internal
     constructor(options: IRushConfigurationProjectOptions);
     get consumingProjects(): ReadonlySet<RushConfigurationProject>;
-    get cyclicDependencyProjects(): Set<string>;
+    get decoupledLocalDependencies(): Set<string>;
```

Comment:
This is a breaking API change. It'd be preferable to keep `cyclicDependencyProjects` around (just return `decoupledLocalDependencies`), but with a `@deprecated` tag.

Reply:
@iclanton I have made the suggested fixes 👍